spec/models/activity_type_spec.rb

```diff
@@ -26,12 +26,12 @@
 
     it 'updates original name so search still works' do
```

Comment:
This is testing that we were still backfilling the original `name` attribute via `copy_searchable_attributes` because the previous version used pg_search. 

We don't really need this behaviour any more. So I think we should either remove that behaviour in this PR or do a follow-on PR to remove it.

Reply:
@ldodds fyi i've removed the non null constraint from intervention types in this pr (see above) so we can write the search content test.